tests/fish_title.test.fish

```diff
@@ -7,18 +7,12 @@ function setup
     mkdir -p /tmp/current/directory/
     cd /tmp/current/directory/
 
-    function _pure_parse_directory; echo /tmp/current/directory; end
 end; setup
 
-function teardown
-    functions --erase _pure_parse_directory
-end
-
-
 @test "fish_title: contains current directory and previous command" (
     set --universal pure_symbol_title_bar_separator '-'
     fish_title 'last-command'
-) = "directory: last-command - fish"
+) = "/tmp/current/directory: last-command - fish"
```

Comment:
I'm unclear why we wanted that behavior. I think it's better to have a consistent path

Reply:
Yep, a consistent behaviour is better, with your feature we can achieve a similar result that reduce the noise from the path.